client/src/components/App.jsx

```diff
@@ -18,7 +22,7 @@ class App extends Component {
         <CostToShip />
         <DeliverTo />
         <EstimatedDelivery />
-        <From />
+        <From city={id2.ship_from_city} state={id2.ship_from_state} />
```

Comment:
it makes perfect sense. Have you tried using a faker? Maybe this way you would have a better chance of getting different States and cities. This is a friendly suggestion

Reply:
I have fake data that is pulled from my db - I don't think this is the most up to date file - it says outdated at the top. The most up to date version gets this from state and pulls the information based on the the product id in the url